sources/quattroshapes.json

```diff
@@ -5,5 +5,7 @@
 	"prefix": "qs",
    	"key": "id",
 	"url": "http://www.quattroshapes.com/",
-	"license": "https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md"
+	"license": "https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md",
+	"description": "A gazetteer of non-overlapping, authoritative polygons around a curated list of places.",
```

Comment:
For the description here please add: 

`Quattroshapes includes open data from government and other sources, many of which require attribution. See [remarks](quattroshapes_remarks.md) for details.`